hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -2367,4 +2373,170 @@ final Stream<DynamicTest> createHollowAccountWithNftTransferAndCompleteIt() {
+                            .andAllChildRecords()
+                            .assertingNothingAboutHashes();
+                    allRunFor(spec, hollowCreate, hapiGetTxnRecord);
+                    if (!hapiGetTxnRecord.getChildRecords().isEmpty()) {
+                        final var newAccountID = hapiGetTxnRecord
+                                .getFirstNonStakingChildRecord()
+                                .getReceipt()
+                                .getAccountID();
+                        spec.registry().saveAccountId(hollowAccountKey, newAccountID);
+                    }
+                    // Verify maxAutomaticAssociations is set to 0
+                    var getInfo = getAccountInfo(hollowAccountKey)
+                            .hasAlreadyUsedAutomaticAssociations(0)
+                            .has(accountWith().key(hollowAccountKey))
+                            .hasMaxAutomaticAssociations(0)
+                            .exposingIdTo(id -> spec.registry().saveAccountId(hollowAccountKey, id));
+
+                    // Delete the account
+                    var delete =
+                            cryptoDelete(hollowAccountKey).sigMapPrefixes(uniqueWithFullPrefixesFor(hollowAccountKey));
+                    allRunFor(spec, getInfo, delete);
+
+                    // Create hollow account with the deleted account alias
+                    var hollowCreate2 = cryptoTransfer((s, b) -> b.setTransfers(TransferList.newBuilder()
+                                    .addAccountAmounts(aaWith(treasuryAlias.get(), -2 * ONE_HBAR))
+                                    .addAccountAmounts(aaWith(hollowAccountAlias.get(), +2 * ONE_HBAR))))
+                            .payingWith(TREASURY)
+                            .signedBy(TREASURY)
+                            .via(transferHBARSToHollowAccountTxn);
+
+                    // Verify new hollow account is created and has no associations
+                    var getInfo2 = getAliasedAccountInfo(hollowAccountKey)
+                            .has(accountWith().hasEmptyKey())
+                            .hasAlreadyUsedAutomaticAssociations(0)
+                            .hasMaxAutomaticAssociations(-1)
+                            .exposingIdTo(id -> spec.registry().saveAccountId(hollowAccountKey, id));
+
+                    // Sends HBAR from hollow account
+                    var hollowAccountTransferHBAR = cryptoTransfer(
+                                    movingHbar(ONE_HBAR).between(hollowAccountKey, TREASURY))
+                            .payingWith(hollowAccountKey)
+                            .signedBy(hollowAccountKey, TREASURY)
+                            .sigMapPrefixes(uniqueWithFullPrefixesFor(hollowAccountKey))
+                            .via(transferHBARSToHollowAccountTxn);
+
+                    allRunFor(spec, hollowCreate2, getInfo2, hollowAccountTransferHBAR);
```

Comment:
Here we are missing one operation that verifies if the hollow accounts is completed and has the correct maxAutoAssociations